src/openshift/components/CryostatContainer.tsx

```diff
@@ -28,14 +28,19 @@ import { ReportService } from '@app/Shared/Services/Report.service';
 import { TargetsService } from '@app/Shared/Services/Targets.service';
 import { pluginServices } from '@console-plugin/services/PluginContext';
 import { Observable } from 'rxjs';
+import CryostatSelector from './CryostatSelector';
+
+export type CryostatService = {
+  name: string;
+  namespace: string;
+};
 
 export const pluginContext: CryostatContext = {
   url: (path?: string): Observable<string> => pluginServices.plugin.proxyUrl(`upstream/${path}`),
   headers: () => {
     const headers = new Headers({
-      // TODO populate these with context selections, maybe taken from redux or localstorage
-      'CRYOSTAT-SVC-NS': '',
-      'CRYOSTAT-SVC-NAME': '',
+      'CRYOSTAT-SVC-NS': localStorage.getItem('cryostat-svc-ns') || '',
+      'CRYOSTAT-SVC-NAME': localStorage.getItem('cryostat-svc-name') || '',
```

Comment:
I think [Session Storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) would actually be a better fit here than Local Storage. It would avoid the problems I alluded to where the user opens two different Console browser tabs at the same time. The user will lose their selection between closing and reopening the console, but I don't think that's so bad of a problem.

Reply:
Also, could you extract the storage keys here to constants?

Reply:
Yeah sure I'll tidy that up